cmd/desync/extract_test.go

```diff
@@ -154,3 +154,44 @@ func TestExtractWithInvalidSeeds(t *testing.T) {
 		})
 	}
 }
+
+func TestExtractWithNonStaticSeeds(t *testing.T) {
+	outDir, err := ioutil.TempDir("", "")
+	require.NoError(t, err)
+	defer os.RemoveAll(outDir)
+	out := filepath.Join(outDir, "out")
```

Comment:
This is in line with the rest of the code, but we could start using https://pkg.go.dev/testing#T.TempDir which was introduced in 1.15.

Reply:
Yeah that's definitely useful. In the latest revision I started to use it.